assignment.js

```diff
@@ -32,6 +32,10 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+for (let i = 0; i < 25; i = i+2) {
+  console.log(i)
+}
+
```

Comment:
@thetripler Good job using the for loop to arrive at your solution. Interesting way to change the value by two as you loop through the values up to 25. One suggestion since your creating a loop you would want to do all of your logic inside the loop and you would want the counter to iterate through each value using `i++`.
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L31)